### ISSUE TITLE
Symbol’s function definition is void: uninstallable

### ISSUE THREAD

**Issue by Trevoke** [doublep/eldev#49]
Hi! Thanks for the work on this project. I need your help figuring out what's wrong with my setup on this box. 

I am using eldev 0.9.2 .

I was working on my laptop, where this project and eldev 0.9.2 work fine, but on my desktop I run into this issue and I can't figure out why. For reference, this is the project: https://github.com/Trevoke/org-gtd.el/

When I run `eldev -d test`, I get the following: 

```
Debugger entered--Lisp error: (void-function uninstallable)
  uninstallable(nil nil)
  eldev--install-or-upgrade-dependencies(project (test) nil nil t nil nil)
  eldev-load-project-dependencies(test)
  eldev-test()
  apply(eldev-test nil)
  eldev-cli(("-d" "test"))
  (kill-emacs (eldev-cli (append (cdr (member "--" command-line-args)) nil)))
  eval((kill-emacs (eldev-cli (append (cdr (member "--" command-line-args)) nil))) t)
  command-line-1(("--execute" "(let ((eldev--emacs-version (format \"%s.%s\" emacs-..." "--execute" "(kill-emacs (eldev-cli (append (cdr (member \"--\" c..." "--" "-d" "test"))
  command-line()
  normal-top-level()
```

This is what my Eldev file looks like:

```
; -*- mode: emacs-lisp; lexical-binding: t; no-byte-compile: t -*-

;; Autodetermined by `eldev init'.
(eldev-use-package-archive 'gnu)
(eldev-use-package-archive 'melpa)

(setq eldev-test-framework 'buttercup)

(eldev-add-extra-dependencies 'test 'with-simulated-input)
```

**Comment by Trevoke** [comment]
*sigh* never mind, I found the global eldev cache, cleared it, and this went away. Sorry about bothering you :)

**Comment by doublep** [comment]
No problem. It actually looks like the bug I fixed in the latest version, 0.9.2. Please have a look at the [release notes](https://github.com/doublep/eldev/releases/tag/0.9.2) and use the advice there. I guess you could run into problems using `upgrade-self` later otherwise.

**Comment by Trevoke** [comment]
Oh, OK. It's interesting because I upgraded from 0.7.2 to 0.9.2 on both
computers and only ran into the issue on one of them, but it
definitely looks like that was the bug... And I found the correct fix.
Thanks!
